ops/fs/bundle/chown.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-import { sendSync, sendAsync } from "./dispatch_json.ts";
-import * as dispatch from "./dispatch.ts";
+import { JsonOp } from "deno_dispatch_json";
+import { opNamespace } from "./namespace.ts";
+
+const OP_CHOWN = new JsonOp(opNamespace, "chown");
```

Comment:
The opNamespace (which is always "builtin" seems uncessary), maybe export a function that wraps this:

    const OP_CHOWN = builtinOp("chown")
    // which also allows (if one op in the file)
    const { sendSync, sendAsync } = builtinOp("chown")

Reply:
I'm going to make `opNamespace` configurable. If I wanted to include multiple versions/instances of the same ops in a project I should be able to.